src/share/classes/com/sun/javatest/regtest/tool/i18n.properties

```diff
@@ -259,6 +259,7 @@ help.main.lock.desc=Lock file to use for tests in "exclusive access" directories
 help.main.nativepath.arg=<path>
 help.main.nativepath.desc=Path to location of native libraries and programs needed \
     by the tests.
+help.main.ve.desc=Verify contents of exclude files
```

Comment:
"validate" may be a better word than "verify"

Reply:
done